report-service/src/main/java/gov/cdc/dataingestion/nbs/services/NbsRepositoryServiceProvider.java

```diff
@@ -51,11 +55,19 @@ public boolean saveXmlMessage(String msgXml) {
 		item.setSpecimenCollDate(null);
 		item.setOrderTestCode(ORDER_TEST_CODE);
 		item.setObservationUid(null);
-    	
-    	nbsInterfaceRepo.save(item);
 
+    	nbsInterfaceRepo.save(item);
 		log.info("Persisted xml to nbs database");
 
     	return true;
     }
+
+	private long getGmtTimestamp() {

```

Comment:
Why GMT zoned time? Where is this coming from?

UTC is the same but more common these days.